### DISCUSSION TITLE
Installing docusaurus on shared cPanel Hosting

### DISCUSSION THREAD

**Discussion by eshanized** [facebook/docusaurus#9942] [Q&A]
Hello There,
I am trying to install this on shared cpanel. My server Config:
![image](https://github.com/facebook/docusaurus/assets/148610067/139fb25e-3e61-4542-b4d8-dc7f6560d651)
Available Software:
![image](https://github.com/facebook/docusaurus/assets/148610067/24d36008-8c89-467f-96df-5b9a9dcce281)
Then,
![image](https://github.com/facebook/docusaurus/assets/148610067/88730e11-f3dc-44e6-808a-c78fc8c1403f)

After what procedure I need to follow?
Thanks in advance.

**Comment by slorber** [comment]
> After what procedure I need to follow?

We don't know what cPanel is and can't recommend a procedure for deploying to this solution.

Read your host official documentation to understand how to deploy a static site (what Docusaurus is)

Or try to [lookup the information on Google](https://www.google.com/search?q=deploy+static+site+to+cpanel&sca_esv=4b3e726407594cc1&rlz=1C5CHFA_enFR1093FR1093&sxsrf=ACQVn08QgdGLFOVHtU1OYhlNEH0ihxaSLw%3A1710244154034&ei=OkHwZf7NAYufkdUP5YqzgAg&ved=0ahUKEwj-0p760-6EAxWLT6QEHWXFDIAQ4dUDCBA&uact=5&oq=deploy+static+site+to+cpanel&gs_lp=Egxnd3Mtd2l6LXNlcnAiHGRlcGxveSBzdGF0aWMgc2l0ZSB0byBjcGFuZWwyBRAhGKABSNggUPgEWIofcAN4AZABAJgBeaABhA-qAQQyNi4yuAEDyAEA-AEBmAIfoALTD8ICChAAGEcY1gQYsAPCAgoQIxiABBiKBRgnwgIEECMYJ8ICERAuGIAEGIoFGJECGMcBGNEDwgILEAAYgAQYigUYkQLCAgoQABiABBiKBRhDwgIOEC4YgAQYxwEYrwEYjgXCAgsQLhiABBjHARjRA8ICBRAAGIAEwgIFEC4YgATCAgcQABiABBgKwgIKEAAYgAQYFBiHAsICCBAAGIAEGMsBwgIKEAAYgAQYywEYCsICCxAAGIAEGIoFGIYDwgIGEAAYFhgewgIHECEYChigAZgDAIgGAZAGBJIHBDI5LjKgB8arAQ&sclient=gws-wiz-serp)

If cPanel is able to run a Node program, you could try to self-host with `docusaurus serve` CLI command, although I wouldn't particularly recommend that compared to a real CDN setup.

https://docusaurus.io/docs/deployment#self-hosting

---

We can only advise you to deploy to a popular solutions for static site hosting, such as Vercel or Netlify. It will be easier, performant, and we can help you more easily in case you encounter a problem.

https://docusaurus.io/docs/deployment
